test/recompilation-tests/starknet-deploy/network.json

```diff
@@ -0,0 +1,4 @@
+{
+    "$schema": "../../network.schema",
+    "devnet": true
```

Comment:
Why are some of your tests using devnet and some integrated-devnet? I think it would be better if they all used the same.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/a887920078d9579750037782a35616569bf230ba

https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/07927e4a1adb923e62ab3417cc04e3b6cf7a6762